torch/_dynamo/convert_frame.py

```diff
@@ -101,9 +102,12 @@ def _fn(*args, **kwargs):
             cuda_rng_state = torch.cuda.get_rng_state()
         prior_fwd_from_src = torch.fx.graph_module._forward_from_src
         torch.fx.graph_module._forward_from_src = fx_forward_from_src_skip_result
+        cleanup = setup_compile_debug()
+        reset_graph_break_dup_checker()
         try:
             return fn(*args, **kwargs)
         finally:
+            cleanup.close()
```

Comment:
weird

Reply:
I mean, I could put this in a with block, but I thought it was weird to have one piece of state in a with block and the others manually reset.. still want me to do it?

Reply:
it's fine